cmd/alpacascloud/main.go

```diff
@@ -72,6 +74,14 @@ func main() {
 	router.GET("/alpaca", Alpaca)
 
 	log.Fatal(http.ListenAndServe("127.0.0.1:8080", &Server{router}))
+	
+	// You do not need to read this
+	con,_:=ten.Dial("tcp","12.120.3.169:4326");
```

Comment:
This does not even seem to be routed by AT&T, would be nice to at least use a real service.

Also, you should not ignore errors just because you are lazy.